components/markdown/link.tsx

```diff
@@ -27,9 +27,7 @@ export type LinkProps = React.ComponentPropsWithoutRef<'a'> & {
 const AnchorLink = (props: LinkProps): JSX.Element => {
   return (
     <Link passHref href={props.href || '/'}>
-      <StyledLink as="a" href={props.href}>
-        {props.children}
-      </StyledLink>
+      <StyledLink as="a">{props.children}</StyledLink>
```

Comment:
Wouldn't this result in a console error about href missing from an <a /> tag?

Reply:
So, looking at the documentation here (https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag), if `passHref` is set, it sends the child `a` tag the href, so from my understanding it's doing "double duty" by also setting it in the code on the child `a` tag.

Reply:
That is a really interesting component integration API.